manifests/input.yaml

```diff
@@ -12,20 +12,20 @@ work/INPUT/JRA55do-drof-ESMFmesh.nc:
     binhash: fa1322dbc0ab951527b165f8327c7b5b
     md5: 3b46260b8c14d12925cf70f82da6658e
 work/INPUT/access-om3-25km-ESMFmesh.nc:
-  fullpath: /g/data/vk83/prerelease/configurations/inputs/access-om3/share/meshes/global.25km/2025.09.04/access-om3-25km-ESMFmesh.nc
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/share/meshes/global.25km/2025.11.27/access-om3-25km-ESMFmesh.nc
   hashes:
-    binhash: 39654bf6d029a845016c98b8e11b1a20
-    md5: fe23759a0a2e99e7b62d5902ec3331e8
+    binhash: 772cf9c667ca5e67a3d2870890e2ea55
+    md5: 0ae5ab11f801a02f2537fb7e1ec02674
 work/INPUT/access-om3-25km-nomask-ESMFmesh.nc:
-  fullpath: /g/data/vk83/prerelease/configurations/inputs/access-om3/share/meshes/global.25km/2025.09.04/access-om3-25km-nomask-ESMFmesh.nc
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/share/meshes/global.25km/2025.11.27/access-om3-25km-nomask-ESMFmesh.nc
   hashes:
-    binhash: ed99d845464a8e4e846b338fc60a15e4
-    md5: ee1487da73ded9d11c3cc009477b3047
+    binhash: bc20c96c3827517fb82b157b1062a886
+    md5: c3cdd8ea948c41e78e55709ebaf4ca91
 work/INPUT/access-om3-25km-rof-remap-weights.nc:
-  fullpath: /g/data/vk83/prerelease/configurations/inputs/access-om3/cmeps/remap_weights/global.25km/2025.09.03/access-om3-25km-rof-remap-weights.nc
+  fullpath: /g/data/vk83/configurations/inputs/access-om3/cmeps/remap_weights/global.25km/2025.11.27/access-om3-25km-rof-remap-weights.nc
   hashes:
-    binhash: 21d1b40f07f6f7b1264a818a2425fc32
-    md5: bad64e2a35312bb66a412c139543a5a9
+    binhash: 39c2a25bec7d738a530ebd6a5fe603ef
+    md5: 80067fb4b5b6a46e483e6d346636b3db
 ? work/INPUT/atmos/3hr/prra/gr/v20240531/prra_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-6-0_gr_195801010130-195812312230.nc
```

Comment:
Apparently the question mark as the beginning of these long paths is an explicit mapping key

https://stackoverflow.com/a/34009604

Did not know that. I guess the yaml library must decide to do this for keys beyond a certain length.

So this is not an error that needs to be rectified, but if anyone knows how we could avoid this I think it would be better as it is a bit unnecessarily confusing.